website/src/components/Chat/ChatConversation.tsx

```diff
@@ -327,7 +327,12 @@ const useAutoScroll = (messages: InferenceMessage[], streamedResponse: string |
     }
 
     messagesEndRef.current?.scrollIntoView({ behavior: "smooth" });
-  }, [messages, streamedResponse]);
+  }, [streamedResponse]);
+
+  useEffect(() => {
+    messagesEndRef.current?.scrollIntoView({ behavior: "smooth" });
+    enableAutoScroll.current = true;
+  }, [messages]);
```

Comment:
Does this happen on submit? or does it run always?

Reply:
I think we should call `messagesEndRef.current?.scrollIntoView({ behavior: "smooth" });` when ever send a new prompt instead.